cmd/snap/cmd_services.go

```diff
@@ -116,3 +123,93 @@ func (s *svcLogs) Execute(args []string) error {
 
 	return nil
 }
+
+type svcStart struct {
+	waitMixin
+	Positional struct {
+		ServiceNames []serviceName `positional-arg-name:"<service>" required:"1"`
+	} `positional-args:"yes" required:"yes"`
+	Enable bool `long:"enable"`
+}
+
+func (s *svcStart) Execute(args []string) error {
```

Comment:
Are those commands tested ? Can probably be done in a similar style as cmd_snap_op_test.go